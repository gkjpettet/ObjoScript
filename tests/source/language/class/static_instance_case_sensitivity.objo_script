class Foo {
  constructor() {}

  static callFoo() {
	print(Foo.foo())
  }

  callFoo() {
	print(this.foo())
  }

  foo() { return "instance foo method" }
  static foo() { return "static foo method" }
}

Foo.callFoo() // expect: static foo method
Foo().callFoo() // expect: instance foo method