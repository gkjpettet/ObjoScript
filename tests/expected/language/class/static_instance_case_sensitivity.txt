static foo method
instance foo method